.gitignore

```diff
@@ -88,3 +88,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# For Kushi's Postgres installation
+kushi_db.sh
```

Comment:
It seems unusual to me to have developer-specific files/names in the .gitignore, but nothing wrong with it in principle. You might want to consider adding it to your own Git repo to keep track of it or, if it is general enough, add it to the bidwire repo itself as a more general db setup script (maybe it could even be an extension of the `setup.sh` script).